plugin-server/src/types.ts

```diff
@@ -756,13 +756,25 @@ export interface EventDefinitionType {
     created_at: string // DateTime
 }
 
+type DateTimePropertyTypeFormat = 'unix_timestamp' | 'YYYY-MM-DD hh:mm:ss' | 'YYYY-MM-DD'
```

Comment:
could also be an enum?

Reply:
I've ended up with 

```
export type UnixTimestampPropertyTypeFormat = 'unix_timestamp' | 'unix_timestamp_milliseconds'
export type DateTimePropertyTypeFormat =
    | 'YYYY-MM-DD'
    | 'YYYY-MM-DDThh:mm:ssZ'
    | 'YYYY-MM-DD hh:mm:ss'
    | 'DD-MM-YYYY hh:mm:ss'
    | 'DD/MM/YYYY hh:mm:ss'
    | 'YYYY/MM/DD hh:mm:ss'
    | 'rfc_822'
export type PropertyTypeFormat = DateTimePropertyTypeFormat | UnixTimestampPropertyTypeFormat
```

Which I think would be clunky as merged enums...?

Reply:
What would it look like as an enum? What I'm imagining is still cleaner than this

Reply:
TS Enums make me sad because they don't have methods. (It's the only bit of Java I miss :))

I've udpated to enum and pushed that. It's not a massive change... The strings are, at least, in one place.